### DISCUSSION TITLE
Using Identify

### DISCUSSION THREAD

**Discussion by Jay123210599** [ImageMagick/ImageMagick#7741] [Help]
### ImageMagick version

7.1.1-27

### Operating system, version and so on

Windows 11

### Description

I have a APNG, along with a lossless animated WEBP and a lossless animated JXL file. Now I want to use ImageMagick's `identify` to see if they have the same quality. How do I do that? How would I know if the quality is either the same or different?

**Comment by snibgo** [comment]
If they are all lossless and hence identical, how can the quality be different?
